packages/now-cli/test/integration.js

```diff
@@ -297,6 +299,30 @@ test('deploy using only now.json with `redirects` defined', async t => {
   t.is(location, 'https://example.com/foo/bar');
 });
 
+test('deploys gatsby and prints cached directories', async t => {
+  const directory = example('gatsby');
+  const packageJsonPath = path.join(directory, 'package.json');
+  const packageJsonOriginal = await readFile(packageJsonPath, 'utf8');
+  const pkg = JSON.parse(packageJsonOriginal);
+
+  async function tryDeploy(cwd) {
+    await execa(binaryPath, [...defaultArgs, '--public', '--confirm'], {
+      cwd,
+      stdio: 'inherit',
+      reject: true,
+    });
+
+    t.true(true);
+  }
+
+  await tryDeploy(directory);
+  await sleep(5000);
+  pkg.scripts.build = `ls -lA && ls .cache && ls public && ${pkg.scripts.build}`;
+  await writeFile(packageJsonPath, JSON.stringify(pkg));
```

Comment:
_Could_ use `writeJson()` from fs-extra, but not a big deal either way.

Reply:
I want to delete `fs-extra` in the future so don't want to add new code using it 😄